src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    @JoinTable(name = "orders_tickets", joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id"))
```

Comment:
Thank you! I employed it solely to ensure that column names in the database maintain a consistent singular form. For instance, without this annotation, names like 'order_id' and 'ticket_id' would adopt a plural form when created by Hibernate, whereas every other table in my database follows a singular naming convention.
Is it ok to use this annotation for this purpose?
![image](https://github.com/mate-academy/hibernate-order-hw/assets/124697095/d0a5186a-6454-4d07-86d0-db0a5b9a7a0d)

Reply:
You don`t need JoinTable because to create one to many relation it is not necessary to create a separate table, if you want to specify custom filed names I suggest using joinColumn